easybuild/easyconfigs/n/ncurses/ncurses-6.0.eb

```diff
@@ -16,9 +16,10 @@ sources = [SOURCE_TAR_GZ]
 
 configopts = [
     # default build
-    '--with-shared --enable-overwrite',
+    '--with-shared --enable-overwrite --with-termlib',
```

Comment:
@ocaisa Why exactly was this required here? It makes this `ncurses` easyconfig that uses `dummy` different from the other `ncurses` easyconfigs, which causes problems (see also https://github.com/easybuilders/easybuild-easyblocks/pull/1088).

I'm not saying we should blindly revert this change, since the split is already taken into account elsewhere (for example in https://github.com/easybuilders/easybuild-easyconfigs/pull/4977/files#diff-03dfb6d80d1923383727d96e696cc53fR20), just wondering what the exact reason was for this.

cc @pforai